syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -254,7 +254,7 @@ def roll(tensor, shifts, **kwargs):
     def handle_func_command(cls, command):
         """
         Operates as a router for functions. A function call always starts
-        by being handled here and 3 scenarii must be considered:
+        by being handled here and 3 scenarios must be considered:
```

Comment:
scenarii is ok - it's when you want to look as a star, you put the italian plural for works that have italian origins